TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Administration/SitesController.php

```diff
@@ -345,9 +345,15 @@ public function createDomainAction(Domain $domain)
      */
     public function deleteDomainAction(Domain $domain)
     {
+        $site = $domain->getSite();
+        if ($site->getPrimaryDomain() === $domain) {
+            $domain->setActive(false);
```

Comment:
Yeah, but I want to use the next active or no domain afterwards as new primary domain.